GOVERNANCE.md

```diff
@@ -0,0 +1,94 @@
+> **Note**
+>
+> This is a **Work in Progress**. Stay tuned for more follow-up updates!
+
+# Governance
+
+TiKV is a community driven project. Anyone with an interest in the project can join the community, contribute to the project design and participate in the decision making process. This document describes how that participation takes place and how to set about earning merit within the project community.
+
+## Code of Conduct
+
+The TiKV community follows the [TiKV Code of Conduct](https://github.com/tikv/tikv/blob/master/CODE_OF_CONDUCT.md). Here are some excerpts:
+
+> In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Community Roles and Responsibilities
+
+Refer to [Community Membership](/community-membership.md).
+
+## Community Structure
+
+![community organization](/media/governance/community_organization.png)
+
+The project is comprised of and operated by the following types of subgroups:
+
+- Project Management Committee, PMC
+- Maintainers
+- Organizer Committee
+- Developer Group
+    - Special Interest Groups, SIGs
+    - Working Groups, WGs
+
+### Project Management Committee
+
+The PMC functions as the core management team that oversees the TiKV community. The PMC has additional responsibilities over and above those of Maintainers. These responsibilities ensure the smooth running of the project. PMC members are expected to review code contributions, participate in strategic planning, approve changes to the governance model and manage the copyrights within the project outputs.
+
+Members of the PMC do not have significant authority over other members of the community, although it is the PMC that votes on new Maintainers or Committers, and makes all major decisions for the future with respect to TiKV, such as project-level governance policies, management of sub-structures, security processes and so on. It also makes decisions when community consensus cannot be reached. In addition, the PMC has access to the project’s private mailing list and its archives.
+
+Membership of the PMC is by invitation from the existing PMC members. A nomination will result in discussion and then a vote by the existing PMC members. PMC membership votes are subject to consensus approval of the current PMC members.
```

Comment:
Should we have a PMC nominated criteria and vote procedure?

Reply:
Yes. That's to be updated.